contracts/Holographer.sol

```diff
@@ -112,6 +112,12 @@ import "./interface/IInitializable.sol";
  * @dev This contract is a binder. It puts together all the variables to make the underlying contracts functional and be bridgeable.
  */
 contract Holographer is Admin, Initializable {
+  bytes32 constant _originChainSlot = 0xd49ffd6af8249d6e6b5963d9d2b22c6db30ad594cb468453047a14e1c1bcde4d;
```

Comment:
Not yet, it's on the todo list. The bridge will be responsible for doing a quick sanity check and make sure chain id did not change since last transaction. This is to prevent a hard fork and duplication of assets due to the fork.